_builder_lib/docsitebuilder/helpers.rb

```diff
@@ -190,6 +202,7 @@ def page(args)
   <link href="#{args[:images_path]}respond.proxy.gif" id="respond-redirect" rel="respond-redirect">
   <script src="#{args[:javascripts_path]}respond.proxy.js" type="text/javascript"></script>
 <![endif]-->
+#{ANALYTICS_SHIM[args[:distro]]}
```

Comment:
So `ANALYTICS_SHIM[args[:distro]]` might become just `args[:analytics_shim]` because you are passing the chosen string from the hash instead of doing the lookup.